Template/Tests/{PROJECT}Tests/{PROJECT}Tests.swift

```diff
@@ -27,5 +27,22 @@ import XCTest
 import {PROJECT}
 
 class {PROJECT}Tests: XCTestCase {
-    
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        //// XCTAssertEqual({PROJECT}().text, "Hello, World!")
+    }
+
+
+}
+
+#if os(Linux)
+extension {PROJECT}Tests {
+    static var allTests : [(String, ({PROJECT}Tests) -> () throws -> Void)] {
+        // FIXME: Should return a list of test case (name, function) tuples
```

Comment:
Ah, sorry - I meant that we can remove the `FIXME` comment. We still need the array.